app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_comment, only: [:update, :destroy]
+
+  def edit
+    @comment = Comment.find(params[:id])
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new comment_params
```

Comment:
Let's use `build` here instead of `new`

Reply:
Could you please always tell us why? 

According to [rails active record explanation here,](https://github.com/rails/rails/blob/959fb8ea651fa6638aaa7caced20d921ca2ea5c1/activerecord/lib/active_record/relation.rb#L83-L85) `build` is just an alias for `new`.